decidim-admin/app/views/decidim/admin/attachments/index.html.erb

```diff
@@ -3,7 +3,7 @@
     <h2 class="card-title">
       <%= t(".attachments_title") %>
       <% if allowed_to? :create, :attachment %>
-        <%= link_to t("actions.new", scope: "decidim.admin", name: t("models.attachment.name", scope: "decidim.admin")), url_for(action: :new), class: "button tiny button--title new" %>
+        <%= link_to t("actions.attachment.new", scope: "decidim.admin", name: t("models.attachment.name", scope: "decidim.admin")), url_for(action: :new), class: "button tiny button--title new" %>
```

Comment:
If we change the locale string as I suggested in the other comment, then we can simplify this to:

```suggestion
        <%= link_to t("actions.attachment.new", scope: "decidim.admin"), url_for(action: :new), class: "button tiny button--title new" %>
```

Because we'd no longer need to pass the name of the resource to create 😄